sdk/typescript/src/types/objects.ts

```diff
@@ -36,6 +36,21 @@ export type SuiMoveObject = {
   has_public_transfer?: boolean;
 };
 
+export type CoinDenominationInfoResponse = {
+  /** Coin type like "0x2::sui::SUI" */
+  coin_type: string;
+  /** min unit, like MIST */
+  basic_unit: string;
+  /** commonly used unit, like SUI */
+  canonical_unit: string;
```

Comment:
nit: I would recommend we remove this for now until we see this diverge from the `symbol` mentioned above.